### ISSUE TITLE
Building Docker Problem

### ISSUE THREAD

**Issue by annessi** [PokemonGoMap/PokemonGo-Map#110]
The building of the docker stops at grunt build. I think is similar to this issue #93 

docker build -t pogomap 
...
```
> pogomap@0.1.0 build /usr/src/app
> grunt build

Running "eslint:src" (eslint) task
Warning: No ESLint configuration found. Use --force to continue.

Aborted due to warnings.

npm ERR! Linux 3.10.35
npm ERR! argv "/usr/bin/node" "/usr/bin/npm" "run-script" "build"
npm ERR! node v6.2.0
npm ERR! npm  v3.8.9
npm ERR! code ELIFECYCLE
npm ERR! pogomap@0.1.0 build: `grunt build`
npm ERR! Exit status 6
npm ERR! 
npm ERR! Failed at the pogomap@0.1.0 build script 'grunt build'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the pogomap package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     grunt build
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs pogomap
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls pogomap
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /usr/src/app/npm-debug.log
INFO[0369] The command [/bin/sh -c apk add --no-cache build-base nodejs  && npm install -g grunt-cli  && npm install  && npm run-script build  && npm uninstall -g grunt-cli  && rm -rf node_modules  && apk del build-base nodejs] returned a non-zero code: 1
```

**Comment by brunoasantos** [comment]
@annessi This should've been fixed with #105

**Comment by annessi** [comment]
Yes now it's working! Thanks
